webrtc.html

```diff
@@ -8901,22 +8907,48 @@ <h2>
               </dt>
               <dd>
                 <p data-tests="RTCRtpSender-getCapabilities.html">
-                  The {{getCapabilities()}} method returns the most optimistic
-                  view of the capabilities of the system for sending media of
-                  the given kind. It does not reserve any resources, ports, or
-                  other state but is meant to provide a way to discover the
-                  types of capabilities of the browser including which codecs
-                  may be supported. User agents MUST support <var>kind</var>
-                  values of <code>"audio"</code> and <code>"video"</code>. If
-                  the system has no capabilities corresponding to the value of
-                  the <var>kind</var> argument, {{getCapabilities}} returns
-                  <code>null</code>.
+                  The static {{RTCRtpSender}}.{{getCapabilities()}} method provides a way to
+                  discover the types of capabilities the user agent supports for sending media
+                  of the given kind, without reserving any resources, ports, or other state.
+                </p>
+                <p>
+                  When the {{getCapabilities}} method is called, the user agent MUST run the
+                  following steps:
+                </p>
+                <ol class=algorithm>
+                  <li class="no-test-needed">
+                    <p>Let <var>kind</var> be the method's first argument.</p>
+                  </li>
+                  <li class="no-test-needed">
+                    <p>If <var>kind</var> is neither `"video"` nor `"audio"` return `null`.</p>
+                  </li>
+                  <li class="no-test-needed">
+                    <p>Return a new {{RTCRtpCapabilities}} dictionary, with its
+                    {{RTCRtpCapabilities/codecs}} member initialized to the
+                    [=list of implemented send codecs=] with <var>kind</var>, and its
```

Comment:
I don't quite like 'with kind'. Could you rephrase that bit here and in the receiver side?
Maybe `with a type of <var>kind</var>`?

Reply:
I've tried to stay consistent with other algorithms with args, using "with" at the call site and "given" in the algorithm heading.

Reply:
Is it a kind or a type?

Reply:
How about "for _kind_"?
```suggestion
                    [=list of implemented send codecs=] for <var>kind</var>, and its
```